src/careamics/model_io/bmz_io.py

```diff
@@ -186,11 +186,15 @@ def export_to_bmz(
         )
 
         # test model description
-        test_kwargs = (
-            model_description.config.bioimageio.model_dump()
-            .get("test_kwargs", {})
-            .get("pytorch_state_dict", {})
-        )
+        test_kwargs = {}
+        if hasattr(model_description, "config") and isinstance(
```

Comment:
Is this covering the case when the Lightning API did not export the config in state dict?

What does this do? Could you comment on it?

I am wondering whether we need to write a warning to Lightning users when they are exporting to BMZ that something is missing (but rn I don't understand this piece of code).

Reply:
@mese79 do you know what this does?